### PR TITLE
Adding category for side dishes

### PR DESCRIPTION
**Pull request by jennank**
Adding missing category for side dishes.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@ ReadTheRecipe
    breakfast/index
    snack/index
    soup/index
+   sides/index
```
Comment (bernardopacini) [inline source/index.rst]: Not a big deal, but what is the ordering here? Should sides go at the bottom?

**Review by bernardopacini — CHANGES_REQUESTED** [review]

File: source/index.rst (hunk 1 above)
Comment (jennank) [reply source/index.rst]: Sounds good to me. I was not sure if we wanted it more as an appetizers/sides or if it should be at the end.

Comment (bernardopacini) [reply source/index.rst]: That makes sense. Do we want an "appetizers" category above soups since soups could be their own "dinner" category?

Comment (jennank) [reply source/index.rst]: I like that idea, appetizers should be separate.

Comment (bernardopacini) [reply source/index.rst]: Okay, sounds good to me. I'll change it to be snack, appetizers, ... main food ..., sides, dessert. Does that sound good?

Comment (bernardopacini) [reply source/index.rst]: I put appetizers right after snacks and put sides right before desserts. We can of course adjust this later if we change our minds.

**Review by bernardopacini — APPROVED** [review]

**Pull request merged**